server.js

```diff
@@ -126,16 +152,32 @@ app.post("/api/schedules", async (req, res) => {
 
     // Generate UUID for new schedule
     const id = uuidv4();
-
-    // Insert schedule into database
     await pool.query(
-      "INSERT INTO schedules (id, namespace, schedule) VALUES ($1, $2, $3)",
-      [id, namespace, schedule],
+      "INSERT INTO schedules (id, schedule, groupName) VALUES ($1, $2, $3)",
+      [id, schedule, groupName],
     );
 
-    scheduleWakeupNamespace({ namespace: namespace, schedule: schedule });
+    let insertedNamespaces = [];
+    for(const namespace of namespaces){
+
+      // Insert schedule into database
+      await pool.query(
+        "INSERT INTO scheduleNamespaces (schedule_id, namespace) VALUES ($1, $2)",
+        [id, namespace],
+      );
```

Comment:
I think both queries should be made as part a transaction, to prevent things from failing in the middle.